content/_releases/23/08.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Product Update - v23.08
+description: Details of the product update v23.08 from 27th of February 2023.
+layout: rel
+version: "23.08"
+section: Product Updates
+category: product-updates
+releaseDate: 2023-02-27
+redirect_from:
+  - /releases/23.08.html
+---
+
+## Features
+
+### Configurable Recipe Use for Contracts
```

Comment:
I think this belongs in tenant / OEM features??

Reply:
I think probably not, because the OEM part is separated with {% if site.data.tenant.name == "elastic.io" %}. That is, not everyone will have access to this section. However, the change applies to everyone.